### PR TITLE
Ozone naming

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable) (N/A)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable) (N/A)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <http://link-to-pr.com>`_) Added feature which does something
- (`#XX <http://link-to-pr.com>`_) Fixed bug identified in (`#XX <http://link-to-issue.com>`_)
```

### CODE REVIEW

File: notebooks/Diagnose-TCR-ECS-TCRE.ipynb [hunk 1]
```diff
@@ -362,164 +434,137 @@
...
+       "MAGICC6       unspecified World  abrupt-2xCO2 not_relevant ppm     Atmospheric Concentrations|CO2  553.488000   \n",
+       "                                                           K       Surface Temperature               2.193655   \n",
+       "                                                           W / m^2 Radiative Forcing                 3.650000   \n",
        "\n",
        "time                                                                                               1898-01-01  \\\n",
-       "climate_model model       scenario     variable                       todo         unit    region               \n",
-       "MAGICC6       unspecified abrupt-2xCO2 Surface Temperature            not_relevant K       World     2.195908   \n",
-       "                                       Radiative Forcing              not_relevant W / m^2 World     3.650000   \n",
-       "                                       Atmospheric Concentrations|CO2 not_relevant ppm     World   553.488000   \n",
+       "climate_model model       region scenario     todo         unit    variable                                     \n",
```
Comment (lewisjared) [inline notebooks/Diagnose-TCR-ECS-TCRE.ipynb:528]: Any idea why the order changed?

File: tests/test_core.py [hunk 2]
```diff
@@ -1636,7 +1636,11 @@ def run(*args, **kwargs):
 def test_empty_output(package):
     package.strict = False
     package.set_output_variables()
-    res = package.run(out_dynamic_vars=[])
+
+    if package.version == 7:
+        res = package.run(out_dynamic_vars=[""])
```
Comment (lewisjared) [inline tests/test_core.py:1641]: Why the empty string vs empty array?

**Review by lewisjared — APPROVED** [review]
LGTM with 2 minor comments

File: tests/test_core.py (hunk 2 above)
Comment (znicholls) [reply tests/test_core.py:1641]: There was some regression in f90nml which meant it wouldn't write properly if I passed an empty list...

File: notebooks/Diagnose-TCR-ECS-TCRE.ipynb (hunk 1 above)
Comment (znicholls) [reply notebooks/Diagnose-TCR-ECS-TCRE.ipynb:528]: I think because we sort the index by default in scmdata now

**Pull request merged**